### ISSUE TITLE
Cannot convert to numpy array.

### ISSUE THREAD

**Issue by setanarut** [alexa-infra/pil-lite#1]
np.array returns PilLite Image object

```
import numpy as np
from PilLite import Image
img = Image.open('img.png')

im = np.array(img)
print(im, im.shape)


<PilLite.Image.Image object at 0x10d3899d0> ()
```

**Comment by alexa-infra** [comment]
Should be fixed by #2

**Comment by hazar-e** [comment]
i can't use use this module instead of pillow, because i can't write animated gif. thanks.
